core/src/test/java/io/questdb/test/cairo/mv/CreateMatViewTest.java

```diff
@@ -902,6 +902,29 @@ public void testCreateMatViewSampleByWithOffset() throws Exception {
         });
     }
 
+    @Test
+    public void testCreateMatViewSetRefreshLimit() throws Exception {
+        assertMemoryLeak(() -> {
+            createTable(TABLE1);
+
+            final String query = "select ts, k, max(v) as v_max from " + TABLE1 + " sample by 30s";
+            execute("CREATE MATERIALIZED VIEW test AS (" + query + ") PARTITION BY WEEK TTL 3 WEEKS;");
+            execute("ALTER MATERIALIZED VIEW test SET REFRESH LIMIT 1m;");
+            drainWalQueue();
```

Comment:
This is a bit on a lower side of testing. Please add cases
- Data is written after the ALTERs
- Other non-structured ALTERs work
- Column add/delete work after this alters

These are scenarios that used to break in the past after adding ALTER operations

Reply:
This test is not aimed to cover functional side of things. `CreateMatViewTest` mostly checks side effects of DDLs, so that's what's done here. As usual with mat views, the functional requirements are covered in the new tests in `MatViewTest`. I'll add tests with other ALTERs interactions there.

Reply:
> * Other non-structured ALTERs work
> * Column add/delete work after this alters

The only other (non-structural) ALTERs for mat views are set symbol capacity and set TTL - I'll add tests for them. Column add/delete is not available for mat views.

Reply:
> I'll add tests with other ALTERs interactions there.

Added a mixed ALTER test in 0fb35d98aebd0a53a76d5b86b821aeb4a737a413